### PR TITLE
Update Mengzhe's PPP

### PR DESCRIPTION
**Pull request by charliemoweng**

### CODE REVIEW

**Comment by felissaf** [comment]
LGTM!

**Review by WangGLJoseph — COMMENTED** [review]
Looking good!

File: docs/team/charliemoweng.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      students under the same module can have the same Telegram handle or email address. (Pull request [#227](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/227))
+    * Fixed the bug that will cause an exception to be thrown when a student is added to a module. (Pull request [#111](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/111))
+    * Wrote tests for Module, Task, and the feature to mark a task as done or undone to increase coverage.
+      (Pull requests [#150](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/150), [#219](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/219), [#224](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/224))
+    * Modified and add on some tests written by others to reflect updates in logic. It was not easy to try to better 
+      the test cases written by others, especially when you are not the one who wrote the logic. I traced the code multiple 
+      times to make sure the test cases that I modified or added will work and improve coverage.
+
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/#breakdown=true&search=charliemoweng)
```
Comment (WangGLJoseph) [inline docs/team/charliemoweng.md:58]: I think this should be a longer link? From after clicking the `</>` icon near your name, then copy that URL

**Review by tanjoewel — APPROVED** [review]
LGTM!

File: docs/team/charliemoweng.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      task under that module and student as done, which means that the student has completed this task.
+    * Justification: TAB aims to make TAs' life easier (thus the name) and by enabling them to mark a task under a 
+      student as done, TAB makes it very straightforward for TAs to keep track of the progress of each student under 
+      them. It takes just a look and some scrolling to know who has done the task and who has not.
+
+
+* **New Feature**: Added in the feature to mark a task under a student as undone (not done yet). (Pull request [#129](https://github.com/AY2122S1-CS2103-F09-1/tp/pull/129))
+    * What it does: It does just the opposite of what the previous feature does - it marks a task under a student as 
+      undone, which means that the student has not completed the task yet.
+    * Justification: We all make mistakes. Students do, TA do, sometimes even professors do. This feature provideds
```
[the remark this answers is not in the document]
Comment (charliemoweng) [reply docs/team/charliemoweng.md:42]: Thanks!

File: docs/team/charliemoweng.md (hunk 1 above)
Comment (charliemoweng) [reply docs/team/charliemoweng.md:58]: ok

**Pull request merged**